runtime/onnion_runtime/equal.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+
+from .error import RunError
+
+
+class Equal:
+    def __init__(self, opset_version, **kwargs):
+        self.version = opset_version
+        self.broadcast = kwargs.get("broadcast")
```

Comment:
`broadcast` and `consumed_inputs` are not found in https://github.com/onnx/onnx/blob/master/docs/Operators.md#Equal

Reply:
`broadcast` is in https://github.com/onnx/onnx/blob/master/docs/Changelog.md#Equal-1.

`consumed_inputs` is my mistake.